backend/FwLite/LcmCrdt/Changes/SetPartOfSpeechChange.cs

```diff
@@ -12,18 +12,18 @@ public override async ValueTask ApplyChange(Sense entity, ChangeContext context)
         if (PartOfSpeechId is null)
         {
             entity.PartOfSpeechId = null;
-            entity.PartOfSpeech = string.Empty;
+            entity.PartOfSpeech = null;
```

Comment:
you can just ignore the PartOfSpeech property as far as this change is concerned, all we're changing is the Id.